packages/terra-menu/src/_MenuWidths.js

```diff
@@ -0,0 +1,10 @@
+const MenuWidths = {
```

Comment:
From how this is used, perhaps this should be an array instead of a hash? I don't see a lot of value of having as a hash atm.